nbclient/client.py

```diff
@@ -329,18 +329,14 @@ def start_kernel_manager(self):
         return self.km
 
     async def _async_cleanup_kernel(self):
+        now = self.shutdown_kernel == "immediate"
```

Comment:
IDK if y'all want to do it like this, but I couldn't see where this was used, and it seemed like it should be here.

Reply:
Yes it is needed in `shutdown_kernel` in jupyter_client. I think it is the right way to do it.